src/pydap/tests/test_handlers_lib.py

```diff
@@ -417,96 +417,82 @@ def setUp(self):
+        self.assertEqual(list(self.data["time_series"]["time"]),
+                         [[10, 21], [15], [], [31, 41, 51, 61]])
 
     def test_selecting_children(self):
         """Test that we can select children."""
-        self.assertEqual(
-            list(self.data[["time_series", "elev"]]), [
-            ([(10, 11, 12), (21, 22, 23)], 1),
-            ([(15, 16, 17)], 4),
-            ([], 9),
-            ([(31, 32, 33), (41, 42, 43), (51, 52, 53), (61, 62, 63)], 16)
-        ])
+        self.assertEqual(list(self.data[["time_series", "elev"]]),
+                         [([(10, 11, 12), (21, 22, 23)], 1),
+                          ([(15, 16, 17)], 4),
+                          ([], 9),
+                          ([(31, 32, 33), (41, 42, 43),
+                            (51, 52, 53), (61, 62, 63)], 16)])
 
     def test_slice(self):
         """Test slicing the object."""
-        self.assertEqual(
-            list(self.data[1::2]), [
-            (2, 4, 4, [(15, 16, 17)]),
-            (4, 8, 16, [
-                (31, 32, 33), (41, 42, 43), (51, 52, 53), (61, 62, 63)])
-        ])
+        self.assertEqual(list(self.data[1::2]),
+                         [(2, 4, 4, [(15, 16, 17)]),
+                          (4, 8, 16, [(31, 32, 33), (41, 42, 43),
+                                      (51, 52, 53), (61, 62, 63)])])
 
     def test_children_data_from_slice(self):
         """Test getting children data from a sliced sequence."""
         self.assertEqual(list(self.data[1::2]["lat"]), [2, 4])
 
     def test_sequence_children_data_from_slice(self):
         """Test getting children data from a sliced sequence."""
-        self.assertEqual(
-            list(self.data[1::2]["time_series"]), [
-            [(15, 16, 17)],
-            [(31, 32, 33), (41, 42, 43), (51, 52, 53), (61, 62, 63)]
-        ])
+        self.assertEqual(list(self.data[1::2]["time_series"]),
+                         [[(15, 16, 17)], [(31, 32, 33), (41, 42, 43),
+                                           (51, 52, 53), (61, 62, 63)]])
 
     def test_deep_slice(self):
         """Test slicing the inner sequence."""
-        self.assertEqual(
-            list(self.data["time_series"][::2]), [
-            [(10, 11, 12), (21, 22, 23)],
-            [],
-        ])
+        self.assertEqual(list(self.data["time_series"][::2]),
+                         [[(10, 11, 12), (21, 22, 23)], []])
 
     def test_integer_slice(self):
         """Test slicing with an integer."""
         self.assertEqual(list(self.data["time_series"][1]), [[(15, 16, 17)]])
 
     def test_filter_data(self):
         """Test filtering the data."""
-        self.assertEqual(
-            list(self.data[self.data["lat"] > 2]), [
-                (3, 6, 9, []),                                                              
-                (4, 8, 16, [(31, 32, 33), (41, 42, 43), (51, 52, 53), (61, 62, 63)]),
-            ])
+        self.assertEqual(list(self.data[self.data["lat"] > 2]),
```

Comment:
I don't think flake8 did a good job w/ this line. You might want to manually fix the spacing.